src/lib/phdi-building-blocks/phdi_building_blocks/standardize.py

```diff
@@ -167,35 +246,104 @@ def standardize_country(
     return standard
 
 
-def standardize_patient_phone(
-    bundle: dict, standardize: Callable = standardize_phone
-) -> dict:
-    """Given a FHIR bundle and a standardization function, standardize the phone numbers
-    in all patient resources in the bundle. By default the standardize_phone function
-    defined in this module is used, but the user is free to provide their own function
-    specifying a standardization process as long as it accepts and returns a string."""
+def phone_truncation_standardization(raw_phone: str, *args) -> str:
```

Comment:
Do we need this function now that we have phone standardization?

Reply:
We're not currently using this functionality in the code, and I can't foresee an instance where we would. I'm going to delete this for now, and should we find that we need it later on we can resurrect it.